apps/L1G/tinder/tinder-frontend/specs/CreatePassword.spec.tsx

```
@@ -1,64 +1,190 @@
+    fireEvent.change(screen.getByPlaceholderText('Password'), { target: { value: 'ValidPass123' } });
+    fireEvent.change(screen.getByPlaceholderText('Confirm password'), { target: { value: 'ValidPass123' } });
+
+    fireEvent.click(screen.getByRole('button', { name: /Continue/i }));
+
+    // This should display the generic server error message
+    expect(await screen.findByText('Something went wrong.')).toBeInTheDocument();
+  });
+
+  it('displays loading text on submit', async () => {
+    // Mock loading state
+    mockSignup.mockImplementation(() => new Promise(() => {})); // Promise never resolves → loading true
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)